### ISSUE TITLE
No such file or directory: 'sudo': 'sudo' (Home Assistant add-on)

### ISSUE THREAD

**Issue by TheGroundZero** [rhasspy/rhasspy#169]
Using the big power button to restart the system instead of the seperate "Restart" button results in this error

Rhasspy add-on on Home Assistant
v2.5.9

```log
[ERROR:2021-01-18 15:16:43,040] rhasspyserver_hermes: [Errno 2] No such file or directory: 'sudo': 'sudo'
Traceback (most recent call last):
  File "/usr/lib/rhasspy/.venv/lib/python3.7/site-packages/quart/app.py", line 1821, in full_dispatch_request
    result = await self.dispatch_request(request_context)
  File "/usr/lib/rhasspy/.venv/lib/python3.7/site-packages/quart/app.py", line 1869, in dispatch_request
    return await handler(**request_.view_args)
  File "/usr/lib/rhasspy/rhasspy-server-hermes/rhasspyserver_hermes/__main__.py", line 2232, in api_system
    proc = await asyncio.create_subprocess_exec(*[str(v) for v in command_list])
  File "/usr/lib/python3.7/asyncio/subprocess.py", line 217, in create_subprocess_exec
    stderr=stderr, **kwds)
  File "/usr/lib/python3.7/asyncio/base_events.py", line 1533, in subprocess_exec
    bufsize, **kwargs)
  File "/usr/lib/python3.7/asyncio/unix_events.py", line 190, in _make_subprocess_transport
    **kwargs)
  File "/usr/lib/python3.7/asyncio/base_subprocess.py", line 37, in __init__
    stderr=stderr, bufsize=bufsize, **kwargs)
  File "/usr/lib/python3.7/asyncio/unix_events.py", line 763, in _start
    universal_newlines=False, bufsize=bufsize, **kwargs)
  File "/usr/lib/python3.7/subprocess.py", line 775, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.7/subprocess.py", line 1522, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'sudo': 'sudo'
[DEBUG:2021-01-18 15:16:43,034] rhasspyserver_hermes: reboot: ['sudo', 'shutdown', '-r', 'now']
[DEBUG:2021-01-18 15:16:40,037] rhasspyserver_hermes: Handling Voices (topic=rhasspy/tts/voices, id=2ffe663f-ad21-4a33-b506-2e29a4e04d20)
```

**Comment by paulvt** [comment]
Yes, it tries to reboot the whole system instead of just a Rhasspy restart. However, it seems that the "sudo" command is not available in the Docker image.
Also I don't think restarting/rebooting should be an option or even work when Dockerized. Maybe Rhasspy could autodetect whether it should show the power button at all. Even if "sudo" exists, you are not assured that "sudo shutdown -r now" will work.

**Comment by RaspiManu** [comment]
I have the same problem with missing sudo rights.

**Comment by synesthesiam** [comment]
OK, I'll hide the button if "sudo" isn't available :+1:

**Comment by Romkabouter** [comment]
@TheGroundZero can you confirm this as fixed in 2.5.11?

**Comment by TheGroundZero** [comment]
> @TheGroundZero can you confirm this as fixed in 2.5.11?

Haven't used Rhasspy in quite some time anymore, so can't confirm nor deny

**Comment by Romkabouter** [comment]
Ok cool. I think it is fixed since I do not see it in my Home Assistant (where no sudo is available)
Feel free to reopen if needed.
